OpenRA.Mods.Common/Traits/Attack/AttackBase.cs

```diff
@@ -31,8 +31,8 @@ public abstract class AttackBaseInfo : PausableConditionalTraitInfo
 		[Desc("Does the attack type require the attacker to enter the target's cell?")]
 		public readonly bool AttackRequiresEnteringCell = false;
 
-		[Desc("Does not care about shroud or fog. Enables the actor to launch an attack against a target even if he has no visibility of it.")]
-		public readonly bool IgnoresVisibility = false;
+		[Desc("Allow firing into the fog to target frozen actors without requiring force-fire.")]
+		public readonly bool TargetFrozenActors = false;
```

Comment:
Doesn't this need an update rule?

Reply:
From above:
> Re lack of update rules: we can't access weapon stats from the actor updates, so theres no way to detect and warn about units that may need to have TargetFrozenActors enabled. I think the best way to handle this is with an entry in the SDK update notes, which is how we have handled similar changes in the past.

Reply:
I see, but I still think we want a rule that removes leftover `IgnoresVisibility` nodes.

Reply:
Done.